client/src/modules/chapters/pages/chapterPage.tsx

```diff
@@ -136,34 +151,56 @@ export const ChapterPage: NextPage = () => {
         {user &&
           (loadingChapterUser ? (
             <Spinner />
-          ) : dataChapterUser ? (
-            <HStack>
-              {dataChapterUser.chapterUser.subscribed ? (
-                <HStack>
-                  <CheckIcon />
-                  <Text>
-                    {dataChapterUser.chapterUser.chapter_role.name} of the
-                    chapter
-                  </Text>
-                  <Button onClick={() => chapterSubscribe(false)} size="md">
-                    Unsubscribe
-                  </Button>
-                </HStack>
-              ) : (
-                <Button
-                  colorScheme="blue"
-                  onClick={() => chapterSubscribe(true)}
-                  size="md"
-                >
-                  Subscribe
-                </Button>
-              )}
+          ) : (
+            dataChapterUser?.chapterUser?.subscribed && (
+              <HStack>
+                {dataChapterUser.chapterUser.subscribed ? (
+                  <HStack justifyContent={'space-between'} width={'100%'}>
+                    <Text fontWeight={500}>
+                      Unfollow upcoming chapter&apos;s events
+                    </Text>
+                    <Button onClick={() => chapterSubscribe(false)} size="md">
+                      Unsubscribe
+                    </Button>
+                  </HStack>
+                ) : (
+                  <HStack justifyContent={'space-between'} width={'100%'}>
+                    <Text fontWeight={500}>
+                      Follow upcoming chapter&apos;s events
+                    </Text>
+                    <Button
+                      colorScheme="blue"
+                      onClick={() => chapterSubscribe(true)}
+                      size="md"
+                    >
+                      Subscribe
+                    </Button>
+                  </HStack>
+                )}
+              </HStack>
+            )
+          ))}
+
+        {user &&
+          (loadingChapterUser ? (
+            <Spinner />
+          ) : dataChapterUser?.chapterUser?.chapter_role ? (
+            <HStack justifyContent={'space-between'}>
+              <Text fontWeight={500}>
+                <CheckIcon marginRight={1} />
+                {dataChapterUser.chapterUser.chapter_role.name} of the chapter
+              </Text>
+              <Button onClick={leaveChapter}>Leave</Button>
             </HStack>
           ) : (
-            <Button colorScheme="blue" onClick={joinChapter}>
-              Join chapter
-            </Button>
+            <HStack>
```

Comment:
```suggestion
            <HStack justifyContent="space-between">
```
For consistency. After joining chapter, buttons are spaced similarly.